repository/spark/docs/latest/resources/dashboards/grafana_spark_operator.json

```diff
@@ -93,7 +93,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "spark_app_submit_count{pod=\"$Spark_Operator_Instance\"}",
+          "expr": "spark_app_submit_count{pod=\"$Spark_Operator_Instance\", job=\"spark-operator-metrics\", endpoint=\"metrics\"}",
```

Comment:
I'd suggest making `job` a parameter, otherwise it will match only `job=spark-operator-metrics`. If this tag is static and doesn't change, then it doesn't make sense to have it. On the other hand, if it is dynamic and changes for different applications, it should be configurable via a parameter.

Reply:
I've removed this tag as it's static here